packages/web-components/fast-foundation/src/horizontal-scroll/horizontal-scroll.ts

```diff
@@ -371,6 +364,8 @@ export class HorizontalScroll extends FoundationElement {
      * @public
      */
     public scrollToNext(): void {
+        this.getRefreshRate();
```

Comment:
Super clever! I'm curious how this part works though. If `this.getRefereshRate()` is effectively async because of rAF, won't fps have the potential to be stale here? Are we just counting on the fact that it will get accurate eventually?

Reply:
I have a fix incoming that will ensure that `this.getRefreshRate()` completes first, which should fix the stale value problem.